src/operators/__tests__/operators.test.ts

```diff
@@ -55,5 +72,18 @@ describe('class.operators', () => {
                 },
             });
         });
+        it('picks a subset of fields to transform', () => {
+            const plain = {
+                bar: 'bar',
+                baz: 'baz',
+            };
+            expect(plainToClass(Foo, plain)).toEqual({
```

Comment:
Notice that the base class (`Foo`) transform all of the input, but the derived class (`Bar`) only **picks** `baz`.